cmd/snap/cmd_sign.go

```diff
@@ -37,11 +40,13 @@ of the assertion can be specified through a "body" pseudo-header.
 `)
 
 type cmdSign struct {
+	clientMixin
 	Positional struct {
 		Filename flags.Filename
 	} `positional-args:"yes"`
 
 	KeyName keyName `short:"k" default:"default"`
+	Chain   string  `long:"chain" optional:"true" optional-value:"remote" choice:"remote" choice:"direct" choice:"local"`
```

Comment:
I wonder if we should follow known and have separate --remote/--direct flags vs --chain. I'm not super sure "local" make sense

Reply:
I can split if you think that is best. Was the idea that there would be a separate flag so it would be

`snap sign --chain --direct`

or that  `--direct/--remote` would completely replace `--chain` so it would be

`snap sign --direct`?

`local` was just to mirror `snap known`. I can come up with an extremely contrived airgapped case where you want to sign something and append locally known asserts without hitting the store, and instead make them available by bringing the files over and acking them locally, but by that point it probably just makes more sense to do it manually, so I'll drop `local` once we have a final call on the semantics split.

Reply:
I meant the former:

> `snap sign --chain --direct`

Reply:
Done